src/services/teams/hooks/index.js

```diff
@@ -794,7 +794,7 @@ exports.beforeAdmin = {
 		authenticate('jwt'),
 		isAdmin,
 		existId,
-		filterToRelated([], 'params.query'),
+		//filterToRelated([], 'params.query'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5531518)